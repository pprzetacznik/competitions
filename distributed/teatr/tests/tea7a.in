98348169 2 93 -1 1000000 6431918673823488789
