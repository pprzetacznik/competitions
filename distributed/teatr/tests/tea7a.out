4614437077656162
